.github/DISCUSSION_TEMPLATE/d-console-error-or-warning.yml

```diff
@@ -0,0 +1,17 @@
+title: "[Console Error/Warning] "
+labels: ["waiting for support"]
+body:
+- type: markdown
+  attributes:
+    value: |
+      Please check your log (`logs/latest.log`) for errors from DiscordSRV, during startup. If you cannot find your errors please post your log here. If you were able to find errors regarding the plugin enabling, please post them here.
```

Comment:
Copy/paste fail?

Reply:
Not sure what you mean

Reply:
It’s a generic error/warning option but the prompt is asking for errors at startup